### PR TITLE
add instructions to customize via CDN

### PR DESCRIPTION
**Pull request by DivineDominion**
Tried `littlefoot({...})` at first but that didn't work :)

### CODE REVIEW

**Comment by goblindegook** [comment]
The example in the README uses `littlefoot.default()`, have you experienced issues with it? (Both default and named export should be supported, by the way.)

File: README.md [hunk 1]
```diff
@@ -124,6 +124,8 @@ performance.
    </script>
    ```
 
+To customize, use `littlefoot.littlefoot()` instead and pass the settings there.
```
Comment (goblindegook) [inline README.md:127]: If `default` doesn't work, I suggest replacing `littlefoot.default()` in the example above with a comment. Something like:

```
littlefoot.littlefoot() // Pass any littlefoot settings here.
```

**Comment by DivineDominion** [comment]
@goblindegook `littlefoot.default()` works, but then you only get the default settings and cannot pass customization parameters in. `littlefoot.littlefoot({ ... })` does work.

**Comment by goblindegook** [comment]
The naming might be throwing you off, `default()` does not mean it's an initialiser that uses the default settings, but rather a consequence of having a function for a default export with CommonJS. You can call `littlefoot.default({ ... })` with whatever settings you choose, it should work the same as `littlefoot.littlefoot({ ... })`.

For that reason, I think your suggestion makes sense and it's less confusing for people.

File: README.md [hunk 2]
```diff
@@ -120,10 +120,12 @@ performance.
      type="application/javascript"
    ></script>
    <script type="application/javascript">
-     littlefoot.default()
+     littlefoot.littlefoot()  // Pass any littlefoot settings here.
    </script>
    ```
 
+If you want to use the default options, use `littlefoot.default()` instead.
```
Comment (goblindegook) [inline README.md:127]: I would remove this line, the comment seems to indicate the functions are different when they are not. `littlefoot()` works for all cases and is less confusing.

**Comment by DivineDominion** [comment]
Sure, there you go!

**Comment by goblindegook** [comment]
Merging. Thank you!

**Pull request merged**